packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -147,7 +147,7 @@ export class OdspDelayLoadedDeltaStream {
 			const websocketTokenPromise = requestWebsocketTokenFromJoinSession
 				? // eslint-disable-next-line unicorn/no-null
 				  Promise.resolve(null)
-				: this.getWebsocketToken!(options);
+				: this.getWebsocketToken(options);
```

Comment:
improved narrowing removed the new for this non-null assertion which thus had to be removed due to our compiler settings.